cnames_active.js

```diff
@@ -1271,6 +1271,7 @@ var cnames_active = {
   "graphics2d": "keyten.github.io/Graphics2D", // noCF? (don´t add this in a new PR)
   "graphql-pokemon": "favware.github.io/graphql-pokemon",
   "graphstore": "besync.github.io/graphstore",
+  "greenenergy": "amchandrakanta.github.io/greenenergy"
```

Comment:
```suggestion
  "greenenergy": "amchandrakanta.github.io/greenenergy",
```
As the CI error notes, your entry is missing a comma at the end of the line.